heon/generate_flow_seq.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import random
+
+
+def flush_file(file):
+    f = open(file, "w")
+    f.write("")
+    f.close()
+
+
+def write_flow_num_to_file(file, value):
+    f = open(file, "a")
+    f.write(f"{value}\n")
+    f.close()
+
+
+def run_single_iteration(flow_num_list, file_to_write):
```

Comment:
Another way to do this would be to shuffle a copy of the list in place and output the (shuffled) list.
But this is also fine.

Reply:
Nice, I never knew that!